### DISCUSSION TITLE
Open collective project and goals: Hardware

### DISCUSSION THREAD

**Discussion by Aggror** [stride3d/stride#1206] [Contributors]
This discussion is for inventorying what kind of (Apple) hardware is required for @xen2 so that he can test out work for macOs.

1. Which hardware do your require
2. What would be the estimated costs?

Any specific text that can be used the open collective project page is welcome.

**Comment by xen2** [comment]
If that's OK, I would prefer to get that only once shader system rewrite is well underway (when we want to start testing Metal, etc.).

I would then need two things:
1. Mac host. still hesitating between Mac Mini or a Macbook Air/Pro (used one might be OK; I need to check if M1 is better or worst than i5/i7 for this kind of thing).
2. iPhone: relatively recent (used probably fine too)

**Reply by manio143** [reply]
I believe there was some cloud offering for testing mobile apps (https://testcloud.xamarin.com/), it's probably changed now a bit, but we could see if they offer iOS testing on it. Probably not great for making small fixes during development but could become part of regression testing.

**Reply by xen2** [reply]
Now that you say it, I remember using macincloud.com for macOS bringup.

**Reply by Aggror** [reply]
So this can remain open for the time being?

**Reply by xen2** [reply]
Yes

**Comment by SupaFabulus** [comment]
I currently have a MacBook Air (M2, 24GB), and I would like to help test macOS support for Stride.
